docs/edge/introduction.rst

```diff
@@ -0,0 +1,344 @@
+Console.
+
+In the CrateDB Cloud Console you can now deploy a cluster from within your
+custom Edge region. Go to the Regions tab of the Organization overview to find
+your custom region and deploy your cluster from there. This will take you to
+the cluster configuration screen.
+
+
+.. _edge-config:
+
+Configure the cluster
+---------------------
+
+
+Configuration
+'''''''''''''
+
+Next, go through the cluster configuration process. You will see your custom
+Edge region is selected, so no region selection is necessary. You can move
+directly to the cluster configuration. You can configure your desired hardware
+values for CPU, RAM, and storage per node manually in the panel provided, as
+well as the number of nodes you want in your cluster.
+
+.. image:: ../_assets/img/cloud-edge-config.png
+   :alt: Cluster configuration panels for CrateDB Edge
+
+On the right the cluster scale overview shows the total hardware values for the
+cluster. This is simply the number of nodes you have chosen, multiplied by the
+values per node you have defined.
+
+At the bottom of the deployment screen you can configure your account settings.
+Since you have already created an organization, it does not need to be set
+here. However, you can now define a project that the cluster can be deployed
+in, as well as the cluster name. You also determine the database username and
+password that you can use to access the cluster `Admin UI`_ later on.
+
+.. image:: ../_assets/img/cloud-edge-settings.png
+   :alt: Account settings menu
+
+Note that the cluster name has certain validation requirements: it may contain
+only numbers, letters, and the dash symbol -. It must begin with a letter and
+end with a letter or a number, and must be at least three characters long.
+
+You can also here define the backup location of your CrateDB Edge cluster. You
+have the option of either using the default backup location for CrateDB Cloud,
+which is managed by us, or use a custom backup location that is convenient to
+you. This has to be an S3 bucket or a location with an equivalent
+functionality. In the latter case, you can set the access key and secret here
+as well. You can test the connection as well; keep in mind that you cannot
+proceed with a custom backup location unless the connection to it is
+functional.
+
+Click *Next* at the bottom right to proceed.
+
+
+Billing
+'''''''
+
+Finally, you will be taken to a new screen where you can fill out your billing
+information. Our payment processing is supported by `Stripe`_. At the bottom
+right you can find the cards accepted by Crate.io. When you have filled out the
+necessary information, click *Deploy* below it to deploy your cluster. Do not
+forget to accept financial authorization by ticking the box at the bottom.
+
+.. image:: ../_assets/img/stripe-billing.png
+   :alt: Billing information screen
+
+The payment and billing information you have submitted will be saved in the
```

Comment:
it might be good to re-assure the customer that we don't save the card details anywhere on our infrastructure but rather request them from Stripe. So perhaps something like

```suggestion
The payment and billing information you have submitted will be available to view in the
```